platform/android/java/app/AndroidManifest.xml

```diff
@@ -53,6 +53,10 @@
             android:name="com.oculus.supportedDevices"
             android:value="all" />
 
+        <!-- Enable VR mode on Pico headsets -->
```

Comment:
Same for this, this should be done through the AAR loader plugin

Reply:
Yes, I think this should be moved to the AndroidManifest.xml of the specific loader in godot_openxr_loaders.